components/infobox/commons/infobox_league.lua

```diff
@@ -441,7 +434,7 @@ function League:_setLpdbData(args, links)
 
 	lpdbData = self:addToLpdb(lpdbData, args)
 	lpdbData.extradata = mw.ext.LiquipediaDB.lpdb_create_json(lpdbData.extradata or {})
-	mw.ext.LiquipediaDB.lpdb_tournament('tournament_' .. self.name, lpdbData)
+	mw.ext.LiquipediaDB.lpdb_tournament('tournament_' .. TextSanitizer.tournamentName(self.name), lpdbData)
```

Comment:
wouldn't it be better to sanitize self.name earlier?
so it is clean when used in /custom etc

Reply:
i was thinking of line 226, i.e. right before the widgetInjector call

Reply:
like e.g. the upcoming matches below the infobox can be called there directly if wanted